template.yaml

```diff
@@ -1066,6 +1064,61 @@ Resources:
 
 
   #==============================ROLES=======================================================================
+
+  DeleteDraftPublicationLambdaRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: sts:AssumeRole
+      Path: /
+      Policies:
+        - PolicyName: LambdaDefaults
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                  - logs:CreateLogGroup
+                  - logs:CreateLogStream
+                  - logs:PutLogEvents
+                  - lambda:InvokeFunction
```

Comment:
Maybe the lambda invoke function is not necessary

Reply:
The original SAM policy template added to the the default lambda InvokeFunction role so I think this is necessary.